src/App.js

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import './App.css';
-import Navbar from './components/navbar/Navbar.js';
-import Wall from './components/Wall';
+import React from ' react ';
+import ' ./App.css ';
+import Navbar from ' ./components/navbar/Navbar.js ';
+import Wall from ' ./components/Wall ';
```

Comment:
There was a misunderstanding, I didn't mean to add spaces there. I meant to add a newline between the last `import` and `function App() {`.